docs/specification/draft/basic/utilities/tasks.mdx

```diff
@@ -0,0 +1,808 @@
+          "createMessage": {}
+        },
+        "elicitation": {
+          "create": {}
+        }
+      }
+    }
+  }
+}
+```
+
+### Capability Negotiation
+
+During the initialization phase, both parties exchange their `tasks` capabilities to establish which operations support task-based execution. Requestors **MUST** only augment requests with task metadata if the corresponding capability has been declared by the receiver.
+
+For example, if a server's capabilities include `tasks.requests.tools.call: true`, then clients may augment `tools/call` requests with task metadata. If a client's capabilities include `tasks.requests.sampling.createMessage: true`, then servers may augment `sampling/createMessage` requests with task metadata.
+
+If `capabilities.tasks` is not defined, the peer **MUST NOT** attempt to create tasks during requests.
+
+The set of capabilities in `capabilities.tasks.requests` is exhaustive. If a request type is not present, it does not support task-augmentation.
+
+`capabilities.tasks.list` controls if the `tasks/list` operation is supported by the party.
+
+`capabilities.tasks.cancel` controls if the `tasks/cancel` operation is supported by the party.
+
+### Tool-Level Negotiation
+
+Tool calls are given special consideration for the purpose of task augmentation. In the result of `tools/list`, tools declare support for tasks via `annotations.taskHint`.
+
+This is to be interpreted as a fine-grained layer in addition to capabilities, following these rules:
+
+1. If a server's capabilities include `tasks.requests.tools.call: false`, then clients **MUST NOT** attempt to use task augmentation on that server's tools, even if they declare tool-level support via `annotations.taskHint`.
+1. If a server's capabilities include `tasks.requests.tools.call: true`, then clients consider the value of `taskHint`, and handle it accordingly:
+   1. If `taskHint` is not present or `false`, clients **MUST NOT** attempt to invoke the tool as a task. Servers **SHOULD** return a `-32601` (Method not found) error if a client attempts to do so.
+   1. If `taskHint` is `true`, clients **SHOULD** attempt to invoke the tool as a task. Servers **MAY** return a `-32601` (Method not found) error if a client does not attempt to do so.
+
+## Protocol Messages
+
+### Creating Tasks
+
+Task-augmented requests follow a two-phase response pattern that differs from normal requests:
+
+- **Normal requests**: The server processes the request and returns the actual operation result directly.
+- **Task-augmented requests**: The server accepts the request and immediately returns a `CreateTaskResult` containing task metadata. The actual operation result becomes available later through `tasks/result` after the task completes.
+
+To create a task, requestors send a request with the `task` field included in the request params. Requestors **MAY** include a `ttl` value representing how long from task creation the requestor would like the task to be retained for.
```

Comment:
What happens if `ttl` is not provided? The spec says receivers MAY override the requested TTL, but doesn't specify behavior when it's omitted. 

Should we clarify that receivers apply their own default policy, and perhaps add a SHOULD recommendation that receivers  avoid indefinite retention for resource management?